cmd/all-in-one/all_in_one_test.go

```diff
@@ -20,24 +21,25 @@ import (
 	"github.com/stretchr/testify/require"
 
 	ui "github.com/jaegertracing/jaeger/model/json"
+	"github.com/jaegertracing/jaeger/ports"
 	"github.com/jaegertracing/jaeger/proto-gen/api_v2"
 )
 
 // These tests are only run when the environment variable TEST_MODE=integration is set.
 
 const (
-	host       = "0.0.0.0"
-	queryPort  = "16686"
-	agentPort  = "5778"
-	healthPort = "13133"
-	queryAddr  = "http://" + host + ":" + queryPort
-	agentAddr  = "http://" + host + ":" + agentPort
-	healthAddr = "http://" + host + ":" + healthPort + "/status"
+	host = "0.0.0.0"
 
 	getServicesURL         = "/api/services"
 	getTraceURL            = "/api/traces/"
 	getServicesAPIV3URL    = "/api/v3/services"
-	getSamplingStrategyURL = "/sampling?service=whatever"
+	getSamplingStrategyURL = "/api/sampling?service=whatever"
+)
+
+var (
+	queryAddr    = fmt.Sprintf("http://%s:%d", host, ports.QueryHTTP)
+	samplingAddr = fmt.Sprintf("http://%s:%d", host, ports.CollectorHTTP)
+	healthAddr   = fmt.Sprintf("http://%s:%d/status", host, ports.CollectorV2HealthChecks)
```

Comment:
just a question for my own understanding: why is this the port here being called `CollectorV2HealthChecks` when it is being used in v1 as well?

Reply:
it's not being used with v1:
```
func healthCheckV2(t *testing.T) {
	if os.Getenv("HEALTHCHECK_V2") == "false" {
		t.Skip("Skipping health check for V1 Binary")
	}
```